resources/base_resource.py

```diff
@@ -1,36 +1,39 @@
 """
-General OCP object
+General OCS object
 """
 import logging
+import yaml
+import tempfile
 from ocs.ocp import OCP
-
-from ocs import defaults
+from utility import utils
+from utility import templating
 
 log = logging.getLogger(__name__)
 
 
 class BaseOCSClass(object):
```

Comment:
If we are going to keep only one class, I think we can just rename this as "OCS" and move here https://github.com/red-hat-storage/ocs-ci/tree/master/ocs

Reply:
@vasukulkarni tend to agree